### PR TITLE
Troop stamina - Displayed

### PR DESCRIPTION
**Pull request by KatarinaSundqvist**
Please review whether these changes can be merged into master. 

- Stamina is now shown on the troops page 

- There is a textbox pre-filled with the missing amount of stamina to reach 100%

- There is a button for healing. No functionality is attached to the button yet

- Some text editor formatting

Please do not delete the branch after merging

### CODE REVIEW

File: WarOfEmpires.Models/Empires/TroopModel.cs [hunk 1]
```diff
@@ -36,5 +39,8 @@ public sealed class TroopModel {
         public bool HasUpkeepRunOut { get; set; }
         public bool HasSoldierShortage { get; set; }
         public string Command { get; set; }
+        public int CurrentStamina { get; set; }
+        public int StaminaToFull { get; set; }
+        public int HealMaxAfford { get; set; }
```
Comment (maikelbos0) [inline WarOfEmpires.Models/Empires/TroopModel.cs]: This one can probably also have a better name once you use it?

File: WarOfEmpires.QueryHandlers/Empires/GetTroopsQueryHandler.cs [hunk 2]
```diff
@@ -38,8 +38,11 @@ public sealed class GetTroopsQueryHandler : IQueryHandler<GetTroopsQuery, TroopM
                 MercenaryTrainingCost = _resourcesMap.ToViewModel(Player.MercenaryTrainingCost),
                 WillUpkeepRunOut = !(player.GetTotalResources() + player.GetResourcesPerTurn() * 48).CanAfford(player.GetUpkeepPerTurn() * 48),
                 HasUpkeepRunOut = player.HasUpkeepRunOut,
-                HasSoldierShortage = player.GetSoldierRecruitsPenalty() > 0
+                CurrentStamina = player.Stamina,
+                HasSoldierShortage = player.GetSoldierRecruitsPenalty() > 0,
+                StaminaToFull = 100 - player.Stamina,
+                //HealMaxAfford = ;
```
Comment (maikelbos0) [inline WarOfEmpires.QueryHandlers/Empires/GetTroopsQueryHandler.cs]: Put a TODO so we don't forget

File: WarOfEmpires/Views/Empire/Troops.cshtml [hunk 3]
```diff
@@ -46,6 +46,24 @@
...
+            @Html.LabelFor(m => m.Stamina)
+        </div>
+        <div class="col-2 col-form-label">
+            @Model.CurrentStamina.ToString(StringFormat.Integer)
+        </div>
+        <div class="col-2">
+            @Html.TextBoxFor(m => m.StaminaToFull, new { @class = "form-control" })
+            @Html.ValidationMessageFor(m => m.StaminaToFull, null, new { @class = "small form-text text-danger" })
+        </div>
+        @*TODO: Make the text selected, for quicker entry of the desired stamina to heal*@
```
Comment (maikelbos0) [inline WarOfEmpires/Views/Empire/Troops.cshtml]: Can put this in GUI or something as a general thing

File: WarOfEmpires/Views/Empire/Troops.cshtml [hunk 4]
```diff
@@ -46,6 +46,24 @@
...
+
+    <div class="row">
+        <div class="col-4 col-form-label">
+            @Html.LabelFor(m => m.Stamina)
+        </div>
+        <div class="col-2 col-form-label">
+            @Model.CurrentStamina.ToString(StringFormat.Integer)
+        </div>
+        <div class="col-2">
+            @Html.TextBoxFor(m => m.StaminaToFull, new { @class = "form-control" })
```
Comment (maikelbos0) [inline WarOfEmpires/Views/Empire/Troops.cshtml]: Do you eventually really need both StaminaToFull and uh, the other one? MaxCanAfford? So they will probably be merged somehow later.

File: WarOfEmpires.Models/Empires/TroopModel.cs (hunk 1 above)
Comment (KatarinaSundqvist) [reply WarOfEmpires.Models/Empires/TroopModel.cs]: Added TODO to rename

File: WarOfEmpires/Views/Empire/Troops.cshtml (hunk 3 above)
Comment (KatarinaSundqvist) [reply WarOfEmpires/Views/Empire/Troops.cshtml]: Removed the comment and added a project in GUI

File: WarOfEmpires/Views/Empire/Troops.cshtml (hunk 4 above)
Comment (KatarinaSundqvist) [reply WarOfEmpires/Views/Empire/Troops.cshtml]: Yes, they will be merged somehow, to use the largest one

**Pull request merged**